### ISSUE TITLE
BagIt-Profile-Identifier tag case/capitalization error in sample bags

### ISSUE THREAD

**Issue by tdilauro** [dpscollaborative/btr_bagit_profile#5]
The `BagIt-Profile-Identifier` tag in `bag-info.txt` is improperly capitalized in all of the sample bags. The `i` in "BagIt" is lowercase, but must be uppercase.

**Comment by diamondap** [comment]
This should be fixed in commit 5628827.

**Comment by tdilauro** [comment]
Thanks, @diamondap. Looks good. Validation results are as expected. Closing.

```
>>> Bag: ../btr_bagit_profile/sample_bags/btr_bad_checksums
✓ Serialization validates
✓ Validates against https://raw.githubusercontent.com/dpscollaborative/btr_bagit_profile/master/btr-bagit-profile.json
Bag validation failed: data/netutil/listen.go sha512 validation failed: expected="00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000" found="b598afc318e6d06f54a162e8e43bbc9cb071fcf0ffb3766b719011d8403d01290d6f2d7a9decc504395501f28f6c452c5a4317ee7bd309d4cd597984227d176d"; manifest-sha512.txt sha512 validation failed: expected="aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" found="fc9a6f7dfac6ab9edf60858391d90131671fc18f1ca6c15fe7bd652a5a0059e76c023cfdf325305d659f2d9719a12c3c97b6d9ead8d4bff3177ab5afdfdf7483"


>>> Bag: ../btr_bagit_profile/sample_bags/btr_bad_extraneous_file
✓ Serialization validates
✓ Validates against https://raw.githubusercontent.com/dpscollaborative/btr_bagit_profile/master/btr-bagit-profile.json
Payload-Oxum validation failed. Expected 6 files and 18242 bytes but found 7 files and 18273 bytes


>>> Bag: ../btr_bagit_profile/sample_bags/btr_bad_missing_payload_file
✓ Serialization validates
✓ Validates against https://raw.githubusercontent.com/dpscollaborative/btr_bagit_profile/master/btr-bagit-profile.json
Payload-Oxum validation failed. Expected 6 files and 18242 bytes but found 5 files and 17083 bytes


>>> Bag: ../btr_bagit_profile/sample_bags/btr_bad_missing_required_tags
✓ Serialization validates
✗ Does not validate against https://raw.githubusercontent.com/dpscollaborative/btr_bagit_profile/master/btr-bagit-profile.json
INVALID: "/Users/Shared/src/git/btr_bagit_profile/sample_bags/btr_bad_missing_required_tags: Required tag 'Bagging-Date' is not present in bag-info.txt."


>>> Bag: ../btr_bagit_profile/sample_bags/btr_good_sha256
✓ Serialization validates
✓ Validates against https://raw.githubusercontent.com/dpscollaborative/btr_bagit_profile/master/btr-bagit-profile.json


>>> Bag: ../btr_bagit_profile/sample_bags/btr_good_sha512
✓ Serialization validates
✓ Validates against https://raw.githubusercontent.com/dpscollaborative/btr_bagit_profile/master/btr-bagit-profile.json
```
